docs/topics/api/stats.rst

```diff
@@ -0,0 +1,109 @@
+==========
+Statistics
+==========
+
+.. note:: This API requires :doc:`authentication <auth>`.
+
+The following API endpoints help you get your archived addon statistics.
+
+---------
+Archiving
+---------
+
+We are archiving statistics data that is older than one year.
+
+We currently archive the following data:
+
++---------------------+--------------------+---------------------------------------+
+| name                |     model_name     | description                           |
++=====================+====================+=======================================+
+| update counts       | updatecounts       | How many users updated this addon     |
++---------------------+--------------------+---------------------------------------+
+| download counts     | downloadcounts     | How many users downloaded this addon  |
++---------------------+--------------------+---------------------------------------+
+| theme update counts | themeupdatecount   | How many users downloaded this theme  |
```

Comment:
How many users have this theme installed (same above for update count: "how many users have this addon installed")